### ISSUE TITLE
Abandoned lib

### ISSUE THREAD

**Issue by Gigasz** [terrylinla/react-native-sketch-canvas#120]
Hi,
this lib seems to be abandoned, so I forked it and published on npm with some minor changes to remove warnings and stuff

https://github.com/gigasz/react-native-sketch-canvas
https://www.npmjs.com/package/@gigasz/react-native-sketch-canvas

Everyone is welcome to contribute

**Comment by joshuajamescabrera10** [comment]
can you help me how to put panResponder on canvas ?

**Comment by DanielBoening** [comment]
@jushenicole I did a fork doing that. You can have a look: https://github.com/PROJEKTPRO/react-native-responsive-sketch-canvas
@Gigasz hope its okay to post this here. It's a fork of this repository, but the main purpose is to put the sketch canvas in a responsive container so that you can zoom and drag.

**Comment by ShaMan123** [comment]
@DanielBoening @Gigasz You guys plan to actively maintain your forks?
If so I can PR with a bunch of enhancements and performance improvements

**Comment by DanielBoening** [comment]
@ShaMan123 Yes, we use it in production. PRs are always welcome and I try to review them early.

**Comment by ShaMan123** [comment]
@DanielBoening I see your fork is even with this one... so I'll leave it as is

**Comment by Gigasz** [comment]
> @DanielBoening @Gigasz You guys plan to actively maintain your forks?
> If so I can PR with a bunch of enhancements and performance improvements

I do not plan to maintain, I've made the fork for a company I no longer work with. However I can add you to the project with publishing permissions on npm if you want to

**Comment by ShaMan123** [comment]
Leave it be.
Thanks
